tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -16,6 +16,18 @@ function goHomeJoinNewGame() {
   cy.log('Joined new game successfully');
 }
 
+function validateMatchResult(match, length, p1, p2) {
+  expect(match.player1.id).to.eq(p1);
+  expect(match.player2.id).to.eq(p2);
+  expect(match.startTime).to.be.greaterThan(0);
+  expect(match.games.length).to.eq(length);
+}
+
+function validateGameResult({status, winner}, expectedWinner) {
+  cy.expect(status).to.eq(3);
```

Comment:
Can we import the `GameStatus` in this file and compare to it?
```suggestion
  cy.expect(status).to.eq(GameStatus.FINISHED);
```